### PR TITLE
Added itasa provider

### PR DESCRIPTION
**Pull request by Elettronik**
Added itasa provider,
I wait for any improvement I can make to make this merged

### CODE REVIEW

File: subliminal/providers/itasa.py [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+            if season_re.match(subtitle.find('name').text.lower()):
+                logger.debug('Found season zip id %d - %r - %r',
+                             int(subtitle.find('id').text),
+                             subtitle.find('name').text,
+                             subtitle.find('version').text)
+
+                content = self._download_zip(int(subtitle.find('id').text))
+                if not is_zipfile(io.BytesIO(content)):   # pragma: no cover
+                    if 'limite di download' in content:
+                        raise TooManyRequests()
```
Comment (fernandog) [inline subliminal/providers/itasa.py]: @Elettronik IMHO I would do like this: raise DownloadLimitExceeded('You reached the download limit')``

File: subliminal/providers/itasa.py [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
+
+        # Download the subs found, can be more than one in zip
+        additional_subs = []
+        for sub in subtitles:
+
+            # open the zip
+            content = self._download_zip(sub.sub_id)
+            if not is_zipfile(io.BytesIO(content)):   # pragma: no cover
+                if 'limite di download' in content:
+                    raise TooManyRequests()
```
Comment (fernandog) [inline subliminal/providers/itasa.py]: also here

**Comment by medariox** [comment]
FYI Itasa has shut down its subtitle section permanently: https://www.italiansubs.net/#solong
As a result, this PR can probably be closed.

**Comment by Elettronik** [comment]
As @medariox  wrote Itasa is shutdown, so closing the Pull Request

**Pull request closed without merging**